### ISSUE TITLE
sysprof: wrong file in package

### ISSUE THREAD

**Issue by pullmoll** [void-linux/void-mklive#62]
`xbps-query -Rf sysprof` prints:
```
/usr/''/sysprof2.service
/usr/bin/sysprof
/usr/bin/sysprof-cli
…
```
I think this is some `systemd` will-o'-wisp.

**Comment by maxice8** [comment]
oh lol

**Comment by maxice8** [comment]
we pass -Dsystemdunitdir=''

it doesn't understand that empty means don't install

**Comment by the-maldridge** [comment]
That's entertaining.  @pullmoll I'm assuming this is the reason that sysprof won't install during a dracut run?

**Comment by pullmoll** [comment]
This has been fixed in 5b4f5e54a0794814c372658aff2691ae4f0657ea
